app/models/MainViewParameters.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+final case class MainViewParameters(
+  includeLanguageSwitch: Boolean,
+  includeDeskPro: Boolean
+)
```

Comment:
So the next step of this change is: 
You can either default these to be true or create an apply method that will create a "default" object for you with these set as true. This will follow the same pattern as the default params in main view